src/components/ReviewTransaction.test.tsx

```diff
@@ -145,3 +153,105 @@ describe('ReviewDetailsItem', () => {
+      result:
+        'tokenAndLocalAmountApprox_oneToken, {"tokenAmount":"10.00","localAmount":"10.00","tokenSymbol":"CELO","localCurrencySymbol":"₱"}',
+    },
+    {
+      tokenId: mockCeloTokenId,
+      feeTokenId: null,
+      amount: 10,
+      feeAmount: null,
+      priceUsd: null,
+      title:
+        'returns only a token amount only for send operation if there is no fee and no local price available',
+      result:
+        'tokenAndLocalAmountApprox_oneToken, {"context":"noFiatPrice","tokenAmount":"10.00","localAmount":"","tokenSymbol":"CELO","localCurrencySymbol":"₱"}',
+    },
+    {
+      tokenId: mockCeloTokenId,
+      feeTokenId: mockCeloTokenId,
+      amount: 10,
+      feeAmount: 0.5,
+      priceUsd: '1',
+      title:
+        'returns token and local amounts if send token and fee token are the same and local price is available',
+      result:
+        'tokenAndLocalAmountApprox_oneToken, {"tokenAmount":"10.50","localAmount":"10.50","tokenSymbol":"CELO","localCurrencySymbol":"₱"}',
+    },
+    {
+      tokenId: mockCeloTokenId,
+      feeTokenId: mockCeloTokenId,
+      amount: 10,
+      feeAmount: 0.5,
+      priceUsd: null,
+      title:
+        "returns only a token amount if send token and fee token are the same but they don't have local price",
+      result: 'tokenAmountApprox, {"tokenAmount":"10.50","tokenSymbol":"CELO"}',
+    },
+    {
+      tokenId: mockCusdTokenId,
+      feeTokenId: mockCeloTokenId,
+      amount: 10,
+      feeAmount: 0.5,
+      priceUsd: '1',
+      title:
+        'returns only a local amount if send token and fee token are different but local prices for both are available',
+      result: 'localAmountApprox, {"localAmount":"10.50","localCurrencySymbol":"₱"}',
+    },
+    {
+      tokenId: mockCusdTokenId,
+      feeTokenId: mockCeloTokenId,
+      amount: 10,
+      feeAmount: 0.5,
+      priceUsd: null,
+      title:
+        'returns multiple token amounts if send token and fee token are different and no local prices available',
+      result:
+        'reviewTransaction.totalAmount_mutlipleTokens_noFiatPrice, {"amount1":"10.00","symbol1":"cUSD","amount2":"0.50","symbol2":"CELO"}',
+    },
+  ])('$title', ({ tokenId, feeTokenId, amount, feeAmount, priceUsd, result }) => {
+    const tokenInfo = { ...mockTokenBalances[tokenId], priceUsd } as unknown as TokenBalance
+    const feeTokenInfo = feeTokenId
+      ? ({
+          ...mockTokenBalances[feeTokenId],
+          priceUsd,
+        } as unknown as TokenBalance)
+      : undefined
+    const tokenAmount = new BigNumber(amount)
+    const localAmount = convertTokenToLocalAmount({ tokenAmount, tokenInfo, usdToLocalRate: '1' })
+    const tokenFeeAmount = feeAmount ? new BigNumber(feeAmount) : undefined
+    const localFeeAmount = tokenFeeAmount
+      ? convertTokenToLocalAmount({
+          tokenAmount: tokenFeeAmount,
+          tokenInfo: feeTokenInfo,
+          usdToLocalRate: '1',
+        })
+      : null
```

Comment:
if the local amounts are being passed as props, you could just hard code values in the provided mock data? 

i also wonder if we could simplify this test by providing the whole `token.info` and `feeTokenInfo` objects in the provided mock data...